build_stage1.sh

```diff
@@ -39,7 +39,11 @@ cd ${BuildDir}
 cp -f ${GitDir}/rhel-atomic-rebuild.repo /etc/yum.repos.d/
 yum -y install ostree rpm-ostree glib2 docker libvirt epel-release libgsystem
 
-cp -f ${GitDir}/atomic7-testing.repo /etc/yum.repos.d/
+
+if [ -f ${GitDir}/atomic7-testing.repo ]; then
```

Comment:
If the script is rerun, and the file has already been moved, the script will fail. It's not an issue when we `cp`, because the script can just do it again. Basically, I'm just trying to make it re-runable.